src/test/java/core/basesyntax/service/impl/DataConverterServiceImplTest.java

```diff
@@ -0,0 +1,133 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverterService;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DataConverterServiceImplTest {
+    private static final List<String> operationsValid1 =
+            readFileContents("src/test/resources/operations_valid_1.csv");
+    private static final List<String> operationsValid2 =
```

Comment:
also, if `operationsValid1` and `operationsValid2` are defined for specific cases, better use explicit name, like "OPERATIONS_SINGLE_FRUIT" & "OPERATIONS_MANY_FRUITS"